### ISSUE TITLE
best practise of combine import compressor from "astro-compressor";

### ISSUE THREAD

**Issue by wanghaisheng** [nhristov/astro-compress#434]
i dont know much tech, just found these two,

can we use them both?

**Comment by NikolaRHristov** [comment]
Yes, `astro-compressor` compresses your files while `astro-compress` minifies them. It's generally preferred to use one over the other `astro-compressor` first and then `astro-compress` in your integrations list, although if you choose the latter first it will compress your files and then it will convert them to .brotli or .gz compression (with `astro-compressor`)
